### DISCUSSION TITLE
How to integrate multiple analytics providers (eg Google and Matomo)?

### DISCUSSION THREAD

**Discussion by Guts** [squidfunk/mkdocs-material#3048] [Q&A] [answered]
Hi there,

I'm trying to add analytics from both tools, Google and Matomo, mainly for a transitional time.
I've followed up [the documentation](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#custom-site-analytics) but this is the latest provider referenced into mkdocs.yml which is used.

My configuration can be viewed here: https://github.com/geotribu/website/blob/matomo/mkdocs.yml#L130-L135

NB: I've added an `extra.matomo_url` as a workaround but it's not related to my question.

**Comment by squidfunk** [comment, marked as answer]
Adding multiple providers is not supported, and support is not planned, so customization is the way to go.

**Reply by Guts** [reply]
Thanks for the clear and quick reply.
